pkgs/data/fonts/catamaran/default.nix

```
@@ -0,0 +1,43 @@
+{
+  stdenvNoCC,
+  lib,
+  fetchzip,
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "catamaran";
+  version = "2024-03-02";
```

```suggestion
  version = "0-unstable-2024-03-02";
```

see https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming